src/mixins/map-mixin.js

```diff
@@ -825,16 +825,21 @@ export default function mapMixin(
         const latLonContainer = _map
           .getContainer()
           .getElementsByClassName("latLonCoordinate")[0]
-        latLonContainer.classList.add("visible")
-        latLonContainer.innerHTML = `Lon: ${lon} </br> Lat: ${lat}`
+        if (latLonContainer) {
```

Comment:
Not related with this ticket, but small enough (quite annoying) to include here.